From 190182eee395430c3345f9cd3eb987a304618932 Mon Sep 17 00:00:00 2001
From: Seth Howell <seth.howell@intel.com>
Date: Fri, 11 Oct 2019 07:06:42 -0400
Subject: [PATCH 4/8] config: allow the shared library to be built.

If we want to be able to build the shared objects for the dpdk submodule
alongside spdk, we will need to disable this configuration option.

Change-Id: If42bc59ff2142b773becf3909ba17da7d2f7373e
Reviewed-on: https://review.gerrithub.io/c/spdk/dpdk/+/463029 (spdk-19.05)
Reviewed-on: https://review.gerrithub.io/c/spdk/dpdk/+/471065 (spdk-19.08)
Reviewed-on: https://review.gerrithub.io/c/spdk/dpdk/+/477012 (spdk-19.11)
Signed-off-by: Seth Howell <seth.howell@intel.com>
Signed-off-by: Tomasz Zawadzki <tomasz.zawadzki@intel.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/dpdk/+/2532
Reviewed-by: Jim Harris <james.r.harris@intel.com>
Reviewed-by: Ben Walker <benjamin.walker@intel.com>
Reviewed-by: Darek Stojaczyk <dariusz.stojaczyk@intel.com>
Tested-by: Jim Harris <james.r.harris@intel.com>
---
 config/common_base | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common_base b/config/common_base
index cc6fd70d5..3975103c6 100644
--- a/config/common_base
+++ b/config/common_base
@@ -57,7 +57,7 @@ CONFIG_RTE_ENABLE_LTO=n
 #
 # Compile to share library
 #
-CONFIG_RTE_BUILD_SHARED_LIB=n
+#CONFIG_RTE_BUILD_SHARED_LIB=n
 
 #
 # Use newest code breaking previous ABI
-- 
2.29.2

